articles/vmware/vmw-ref-vcd-91.md

```diff
@@ -48,7 +48,9 @@ You should be aware of the following end-of-life and end-of-support warnings for
 
 ## Known issues
 
-You can only manage advanced gateways in the new vCloud Director 9.1 tenant portal. If you have not yet converted your edge to an advanced gateway, you must first switch to the vCloud Director web console to manage the edge.  For more information about converting your edge to an advanced gateway, see [*How to convert your edge to an advanced gateway*](vmw-how-convert-edge.md). For more information about switching to the web console, see [*How to switch to the vCloud Director web console from the tenant portal*](vmw-how-switch-web-console.md). 
+There are some tasks, including managing standard edges, that are not currently possible using the vCloud Director 9.1 tenant portal. To perform these tasks, you must use the vCloud Director web console. For more information, including a list of tasks that must be performed in the web console, see [*How to switch to the vCloud Director web console from the tenant portal*](vmw-how-switch-web-console.md). For information about converting your edge to an advanced gateway, which can be managed in the tenant portal, see [*How to convert your edge to an advanced gateway*](vmw-how-convert-edge.md).
+
+If you disable your firewall, NAT and Load Balancing features are also disabled. The reason behind this is to enable you to perform testing behind the edge without the risk of compromise from external sources. While this feature was only present in advanced edge gateways on vCloud 8.20, this feature has now been enforced on standard edge gateways in vCloud 9.1. To ensure that you're not affected by any potential issues, we recommend that you review your edge configuration, apply appropriate rules and enable the firewall. Not only will this prevent an outage on your systems after the vCloud upgrade, we also recommend that firewalls should be enabled as part of best practice to reduce attack vectors on your cloud environment. Before you perform any work on the edge, we strongly recommend that you take a backup of the configuration via the API as a restore point. For information about how to do this, see [*Reviewing the edge gateway configuration*](https://docs.ukcloud.com/articles/vmware/vmw-how-change-ipsec-vpn-api.html#reviewing-the-edge-gateway-configuration) in *How to change IPsec VPN settings via the vCloud Director API*.
```

Comment:
Sentence starting Not only will this prevent doesn't quite make sense. 
Suggest rewording